lib/components/narrative/default/default-itinerary.js

```diff
@@ -1,34 +1,76 @@
+import moment from 'moment-timezone'
 import coreUtils from '@opentripplanner/core-utils'
 import React from 'react'
+import { FormattedMessage, FormattedNumber } from 'react-intl'
+import { connect } from 'react-redux'
 
 import NarrativeItinerary from '../narrative-itinerary'
 import ItineraryBody from '../line-itin/connected-itinerary-body'
-import ItinerarySummary from './itinerary-summary'
 import SimpleRealtimeAnnotation from '../simple-realtime-annotation'
-import { getTotalFareAsString } from '../../../util/state'
 
-const { isBicycle, isTransit } = coreUtils.itinerary
-const { formatDuration, formatTime } = coreUtils.time
+import ItinerarySummary from './itinerary-summary'
+
+const { calculateFares, isBicycle, isTransit, isMicromobility } = coreUtils.itinerary
 
+/**
+ * Obtains the description of an itinerary in the given locale.
+ */
 // FIXME move to core utils
 function getItineraryDescription (itinerary) {
```

Comment:
I recommend changing this into a proper functional component:
```suggestion
function ItineraryDescription ({ itinerary }) {
```

Reply:
Just out of curiosity, can you explain the advantages of functional components here? More readable?

Reply:
Even though a generically named function like this can return JSX no problem, having this function named like a component implies an understanding to the developer (at least it does for me) that this method isn't returning simple text or something. Furthermore, by changing the signature to take in an object it can end up reducing the need to refactor the arguments from positional arguments if it is already in a format that a Component is called with.

Reply:
Addressed in https://github.com/opentripplanner/otp-react-redux/pull/375/commits/5e2b6708003925b1163418bd629f3870889ec6a2.